kie-server-parent/kie-server-tests/kie-server-integ-tests-jbpm/src/test/java/org/kie/server/integrationtests/jbpm/ProcessServiceIntegrationTest.java

```diff
@@ -178,6 +189,28 @@ public void testAbortExistingProcess() {
             fail(e.getMessage());
         }
     }
+    
+    @Test()
+    public void testAbortExistingProcessWrongContainer() {
+        Map<String, Object> parameters = new HashMap<String, Object>();
```

Comment:
Explicit arguments are not really needed
```suggestion
        Map<String, Object> parameters = new HashMap<>();
```